src/model/sechubModel.ts

```diff
@@ -31,31 +40,34 @@ export interface FindingModel {
     result: FindingResult
 }
 
-export interface FindingResult{
+export interface FindingResult {
     count: number
-    findings : FindingNode[]
+    findings: FindingNode[]
 }
 
 export interface FindingNode {
     id: number
     name: string
+
+    description: string
```

Comment:
What happens when the description in a report is not set/null?

```suggestion
    description?: string
```